examples/server/bench/script.js

```diff
@@ -0,0 +1,84 @@
+import http from 'k6/http';
+import { check, sleep } from 'k6';
+import { SharedArray } from 'k6/data';
+import { Counter, Gauge, Rate } from 'k6/metrics';
+
+const data = new SharedArray('conversations', function () {
+    return JSON.parse(open('./ShareGPT_V3_unfiltered_cleaned_split.json'))
+
+        // Filter out the conversations with less than 2 turns.
+        .filter(data => data["conversations"].length >= 2)
+        // Only keep the first two turns of each conversation.
+        .map(data => Array(data["conversations"][0]["value"], data["conversations"][1]["value"]));
+});
+
+const llamacpp_prompt_tokens = new Gauge('llamacpp_prompt_tokens');
+const llamacpp_completion_tokens = new Gauge('llamacpp_completion_tokens');
+
+const llamacpp_completions_tokens_seconds = new Gauge('llamacpp_completions_tokens_seconds');
+
+const llamacpp_prompt_tokens_total_counter = new Counter('llamacpp_prompt_tokens_total_counter');
+const llamacpp_completion_tokens_total_counter = new Counter('llamacpp_completion_tokens_total_counter');
+
+const llamacpp_completions_truncated_rate = new Rate('llamacpp_completions_truncated_rate');
+const llamacpp_completions_stop_rate = new Rate('llamacpp_completions_stop_rate');
+
+export const options = {
+    thresholds: {
+        llamacpp_completions_truncated_rate: [
+            // more than 10% of truncated input will abort the test
+            { threshold: 'rate < 0.1', abortOnFail: true, delayAbortEval: '1m' },
+        ],
+    },
+    scenarios: {
+        completions: {
+            executor: 'ramping-vus',
+            startVUs: 1,
+            stages: [
+                {duration: '1m', target: 8},
+                {duration: '3m', target: 8},
+                {duration: '1m', target: 0},
+            ],
+            gracefulRampDown: '30s',
+        },
+    },
+};
+
+export default function () {
+    const conversation = data[0]
```

Comment:
In the beginning of this file, you use have semicolon at the end of each line, but here you don't use it. For consistency, can you either remove all of them or leave them everywhere?